stl/inc/xutility

```diff
@@ -14,6 +14,10 @@
 #include <cstdlib>
 #include <cstring>
 
+#if _HAS_CXX23
+#include <limits>

```

Comment:
No change requested. `numeric_limits<I>::max()` is far more commonly used than other stuffs in `<limits>`, so it seems better to me to have a variable template for it so that we can sometimes avoid including the whole `<limits>`.

Reply:
I'm dealing with this now.